testing/schemaPath1/test_path1.js

```diff
@@ -1,84 +1,87 @@
-		test_returns(input: test_returns_input): test_return
-	}
-
-	type test_result {
-		success : Boolean!
-		message : String
-		books: [test_book]
-		throw : Boolean
-		throw2 : Boolean
-	}
-
-	type test_book {
-		title: String
-		author: String
-	}
-
-	type test_return {
-		data: String
-		nested: test_return
-	}
-	
-	input test_books_filter {
-		throwError : Boolean
-		authError : Boolean
-		message : String
-	}
-	
-	input test_books_withRequiredValues {
-		requiredVal : String!
-		nonRequiredVal : String
-	}
-
-	input test_returns_input {
-		data: String
-		nested: test_returns_input
-	}
-`;
-
-const resolvers = {
-	Query: {
-		test_books(parent, { filter : { throwError = false, authError = false, message } = {} }, context, info) {
-			if (throwError) {
-				throw new Error("Test throw!");
-			}
-			
-			if (authError) {
-				throw new AuthenticationError("Test throw!");
-			}
-			
-			return {
-				success : true,
-				message : message || "messageValue",
-				books : [
-					{
-						title: 'Harry Potter and the Chamber of Secrets',
-						author: 'J.K. Rowling'
-					},
-					{
-						title: 'Jurassic Park',
-						author: 'Michael Crichton'
-					}
-				]
-			}
-		},
-		test_returns(parent, { input }, context, info) {
-			return input;
-		}
-	},
-	test_result : {
-		throw : function() {
-			throw new Error("Thrown from throw");
-		},
-		throw2 : function() {
-			throw new Error("Thrown from throw2");
-		}
-	}
-};
-
-module.exports = new ApolloServer({ typeDefs, resolvers });
\ No newline at end of file
+const { gql, AuthenticationError } = require('apollo-server');
+
+const typeDefs = gql`
+	extend type Query {
+		test_books(filter: test_books_filter, withRequiredValues: test_books_withRequiredValues): test_result
+		test_returns(input: test_returns_input): test_return
+	}
+
+	type test_result {
+		success : Boolean!
+		message : String
+		books: [test_book]
+		throw : Boolean
+		throw2 : Boolean
+	}
+
+	type test_book {
+		title: String
+		author: String
+	}
+
+	type test_return {
+		data: String
+		nested: test_return
+	}
+	
+	input test_books_filter {
+		throwError : Boolean
+		authError : Boolean
+		message : String
+	}
+	
+	input test_books_withRequiredValues {
+		requiredVal : String!
+		nonRequiredVal : String
+	}
+
+	input test_returns_input {
+		data: String
+		nested: test_returns_input
+	}
+`;
+
+const resolvers = {
+	Query: {
+		test_books(parent, { filter : { throwError = false, authError = false, message } = {} }, context, info) {
+			if (throwError) {
+				throw new Error("Test throw!");
+			}
+			
+			if (authError) {
+				throw new AuthenticationError("Test throw!");
+			}
+			
+			return {
+				success : true,
+				message : message || "messageValue",
+				books : [
+					{
+						title: 'Harry Potter and the Chamber of Secrets',
+						author: 'J.K. Rowling'
+					},
+					{
+						title: 'Jurassic Park',
+						author: 'Michael Crichton'

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-graphql-client/pullRequest?prid=9067296)
```suggestion
						author: "Michael Crichton"
```